{ 'cebit_weight_ready': { 'head': [],
                          'left_arm': [ { 'duration': 1.96,
                                          'name': u'cebit_weight_1',
                                          'positions': [ -0.8654,
                                                         0.0188,
                                                         -0.2016,
                                                         0.0235,
                                                         -1.5611,
                                                         0.0342,
                                                         -0.0525],
                                          'starttime': 0.0}],
                          'left_leg': [ { 'duration': 1.96,
                                          'name': u'cebit_upright',
                                          'positions': [ 0.0199,
                                                         -0.0901,
                                                         -0.6282,
                                                         1.2101,
                                                         0.6201,
                                                         -0.0899],
                                          'starttime': 0.0}],
                          'right_arm': [ { 'duration': 1.96,
                                           'name': u'cebit_weight_1',
                                           'positions': [ 0.8654,
                                                          -0.0188,
                                                          0.2016,
                                                          -0.0235,
                                                          1.5611,
                                                          -0.0342,
                                                          0.0525],
                                           'starttime': 0.0}],
                          'right_leg': [ { 'duration': 1.96,
                                           'name': u'cebit_upright',
                                           'positions': [ -0.0199,
                                                          0.0901,
                                                          0.6282,
                                                          -1.2101,
                                                          -0.6201,
                                                          0.0899],
                                           'starttime': 0.0}],
                          'torso': [ { 'duration': 1.96,
                                       'name': u'cebit_weight',
                                       'positions': [0.0001, -0.1883],
                                       'starttime': 0.0}]}}
